reporter/src/funTest/kotlin/reporters/opossum/OpossumReporterFunTest.kt

```diff
@@ -61,8 +61,7 @@ class OpossumReporterFunTest : WordSpec({
 private fun TestConfiguration.generateReport(ortResult: OrtResult): String {
     val input = ReporterInput(
         ortResult = ortResult,
-        resolutionProvider = DefaultResolutionProvider(ortResult.getResolutions()),
-        howToFixTextProvider = { "Some how to fix text." }
+        resolutionProvider = DefaultResolutionProvider(ortResult.getResolutions())
```

Comment:
Commit message nit: Maybe explicitly say that this parameter is irrelevant for the test and could be anything, but for simplicity just stick to the default value.